templates/common/user_tools.html

```diff
@@ -44,6 +44,10 @@ <h2 class="username">
       <ul id="user-actions" class="toolset clear">
 
         $if not relationship['is_self']:
+		  <li>
+		    $if relationship['followby']:
+			  <span class="icon icon-20 icon-followed"></span>Following You
+		  </li>
```

Comment:
Possibly not. The editor I used (and for some reason, GitHub's preview) showed them as 2-space indents, so I never noticed anything off and assumed for the most part that everything was tabbed.
I'm kind of new to using github; is there a way I can change this, or do I have to close the request and make a new one?